src/apple/config/raw.rs

```diff
@@ -42,10 +42,44 @@ impl Display for PromptError {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq)]
+#[serde(untagged)]
+pub enum OneOrMany<T: Debug> {
+    One(T),
+    Many(Vec<T>),
+}
+
+impl<T: Debug> From<OneOrMany<T>> for Vec<T> {
+    fn from(from: OneOrMany<T>) -> Self {
+        match from {
+            OneOrMany::One(val) => vec![val],
+            OneOrMany::Many(vec) => vec,
+        }
+    }
+}
+
+impl<T: Debug> Display for OneOrMany<T> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Self::One(one) => write!(f, "{:?}", one),
+            Self::Many(vec) => write!(f, "{:?}", vec),
```

Comment:
Using the debug formatter in a `Display` impl isn't very displayish

Reply:
hmm, I used it here because it displays it with exactly the formatting we want, and it makes the `collect_str`. I guess the alternative is to wither more explicitly copy the same behavior in the display, or just move that logic to the Serialize impl?